src/Bot/Poll.hs

```diff
@@ -90,6 +91,16 @@ cancelPollCommand Message {messageSender = sender} = do
       say [qms|TwitchVotes The current poll has been cancelled!|]
     Nothing -> replyToSender sender "No polls are in place"
 
+
+-- TODO maybe use this in announcePollResults too?
```

Comment:
Please keep in mind that all of your TODOs are gonna become issue titles after the merge (see [CONTRIBUTING.md] for more info). So please name them a little bit more descriptive to make it easier to scroll through them. I'd recommend something like "maybe use rank in announcePollResults too?", because "this" makes the title too vague.

[CONTRIBUTING.md]: https://github.com/tsoding/HyperNerd/blob/fa1c49722eb6c448da4426d1c745a1c1b1a9a13d/CONTRIBUTING.md#pull-requests